docs/admin/integration/integration.md

```diff
@@ -33,32 +33,23 @@ Set the following environment variables for the `ssi-credential-issuer` to integ
 
-| Name                                  | Description                 | Expected Value                     |
-|---------------------------------------|-----------------------------|------------------------------------|
-| edc.iam.issuer.id                     | IAM Issuer ID               | did:web:localhost:BPNL000000000000 |
-| edc.iam.trusted-issuer.1-issuer.id    | Trusted Issuer ID           | did:web:localhost:BPNL000000000000 |
-| edc.iam.sts.dim.url                   | DIM URL                     | https://localhost/api/sts          |
-| edc.iam.sts.oauth.token.url           | OAuth Token URL             | https://localhost/oauth/token      |
-| tx.edc.iam.iatp.credentialservice.url | IATP Credential Service URL | https://localhost/api              |
-| edc.iam.sts.oauth.client.id           | OAuth Client ID             | BPNL000000000000                   |
-| edc.iam.sts.oauth.client.secret.alias | OAuth Client Secret Alias   | BPNL000000000000                   |
-
-## BPN DID Resolution Service Integration
-
-Set the following environment variables for the `bpn-did-resolution-service` to integrate with the wallet stub
-application:
-
-| Name            | Description     | Expected Value                         |
-|-----------------|-----------------|----------------------------------------|
-| BASE_WALLET_BPN | Base Wallet BPN | BPNL000000000000                       |
-| BASE_WALLET_DID | Base Wallet DID | did:web:localhost:BPNL000000000000     |
-| trustedIssuers  | Trusted Issuers | ["did:web:localhost:BPNL000000000000"] |
+| Name                                  | Description                                  | Expected Value                              |
+|---------------------------------------|----------------------------------------------|---------------------------------------------|
+| edc.iam.issuer.id                     | IAM Issuer ID                                | did:web:localhost:BPNL000000000000          |
+| edc.iam.trusted-issuer.1-issuer.id    | Trusted Issuer ID                            | did:web:localhost:BPNL000000000000          |
+| edc.iam.sts.dim.url                   | DIM URL                                      | https://localhost/api/sts                   |
+| edc.iam.sts.oauth.token.url           | OAuth Token URL                              | https://localhost/oauth/token               |
+| tx.edc.iam.iatp.credentialservice.url | IATP Credential Service URL                  | https://localhost/api                       |
+| edc.iam.sts.oauth.client.id           | OAuth Client ID                              | BPNL000000000000                            |
+| edc.iam.sts.oauth.client.secret.alias | OAuth Client Secret Alias                    | BPNL000000000000                            |
+| tx.edc.iam.iatp.bdrs.server.url       | BDRS server URL, it will be wallet stub host | http://<wallet stub host>//api/v1/directory |
```

Comment:
```suggestion
| tx.edc.iam.iatp.bdrs.server.url       | BDRS server URL, it will be wallet stub host | http://localhost/api/v1/directory |
```
I think it’s better to follow the format by using `localhost`. I assume that when you refer to `<wallet stub host>`, in this case, it means `localhost`. You might consider adding a note at the top of the document to clarify that when you mention `localhost`, you are referring to the `wallet stub host`.

Reply:
Doc updated: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/30/commits/b462af46b620c9cdcce671742f9bd0ac261f31fa